templates/nodejs/config/common.yml

```diff
@@ -16,7 +16,7 @@ default:
       exchange:
       username:
       password:
-      host: {{ asyncapi.server(params.server).url() | host }}
+      host: amqp://{{ asyncapi.server(params.server).url() | replaceVariablesWithValues(asyncapi.server(params.server).variables())  | host }}
```

Comment:
It shouldn't have `amqp://` on the `host`. Hosts don't contain protocol information. You probably have to add `amqp://` where this value is being used (if needed).

Reply:
you are right, works well without it......makes me thing why I added it as I don't remember